fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -873,6 +872,51 @@ public Void visitFunctionCall(FunctionCallExpr node, Scope scope) {
                     throw new SemanticException(fnName + " should have at least two inputs");
                 }
                 fn = Expr.getBuiltinFunction(fnName, argumentTypes, Function.CompareMode.IS_NONSTRICT_SUPERTYPE_OF);
+            } else if (fnName.equals("array_generate")) {
+                if (node.getChildren().size() < 1 || node.getChildren().size() > 3) {
```

Comment:
can array_generate accept only one argument?
if not so,  node.getChildren().size() < 2

Reply:
It can accept only one argument, the output of array_generate(3) is [1,2,3]